index.test.js

```diff
@@ -163,3 +163,9 @@ test('exposes "BroadcastChannel"', () => {
   expect(channel).toBeInstanceOf(BuiltinBroadcastChannel)
   channel.unref()
 })
+
+test('exposes "TransformStream"', () => {
+  expect(globalThis).toHaveProperty('TransformStream')
+  const channel = new TransformStream()
+  expect(channel).toBeInstanceOf(TransformStream)
```

Comment:
For instance comparison, we need to make sure it's the instance of Node's `TransformStream`. For that, it's a good idea to import this class directly from `node:stream/web`, I believe. 

I can add this fix.